documentation/source/getting_started/components_architecture.rst

```diff
@@ -0,0 +1,23 @@
+Components and Architecture
+===========================
+This section includes OpenEBS components and architecture.
+
+Components
+----------
+
+OpenEBS platform contains three major components:
+
+  * Storage PODs or VSMs
+  * An orchestration engine or VSM Scheduler called Maya
+  * The OpenEBS hosts that provide the data store from either local disks or remote disks
+
+.. image:: /_static/basic.png
+
+Architecture
+-------------
+
+Maya is the orchestration engine that schedules the VSMs among OpenEBS hosts as needed. Maya driver (Docker Volume Driver for Maya) plays an important role in achieving the smooth flow of provisioning of VSMs and attaining the application consistent snapshots. The data is kept in more than one copy among the OpenEBS hosts through a backend network replication, thus achieving the necessary redundancy. VSMs expose the iSCSI interface currently.
```

Comment:
We do not have Docker volume driver for Maya

Reply:
Deleted (Docker volume driver for Maya). Changes will reflect when I push the document to the readthedocs website.

Reply:
There is "(Refer to the Roadmap: TBD)" under Architecture section. Where should I refer to?